BitFaster.Caching.UnitTests.Std/BitFaster.Caching.UnitTests.Std.csproj

```diff
@@ -0,0 +1,31 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>net6.0</TargetFrameworks>
+    <LangVersion>9.0</LangVersion>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="coverlet.msbuild" Version="6.0.0">
```

Comment:
If this stomps the code coverage, a simple solution would be to remove coverage collection.

Reply:
Coverage is collected into different directories, so this is not an issue.